addons/spreadsheet_dashboard/static/tests/clickable_cells/clickable_cells.js

```diff
@@ -50,40 +45,41 @@ QUnit.test("Invalid pivot/list formulas should not be clickable", async (assert)
             },
         ],
     };
-    const model = new Model(data, { mode: "dashboard" });
-    const target = await createDashboardWithModel(model);
+    const target = await createDashboardActionWithData(data);
     assert.containsNone(target, ".o-dashboard-clickable-cell");
 });
 
 QUnit.test("pivot/list formulas should be clickable", async (assert) => {
-    const list = functionRegistry.get("ODOO.LIST");
```

Comment:
I had to change the strategy quite a bit here. The naively mocked functions were not working anymore. I tried to properly create the model without mocking anything. But boy I had to fight with `Cannot add '${key}' in this registry: it already exists` :grimacing: it's impossible to instantiate two envs in the same test. Even with all the possible registry cleaning I tried.
Now, the data is given to `createDashboardActionWithData`, we don't need to create a temporary model before